spec/factories.rb

```diff
@@ -52,7 +52,9 @@
     end
 
     after(:build) do |organization, evaluator|
-      create_list(:user, evaluator.users_count, organizations: [organization])
+      if evaluator.users.count < evaluator.users_count
+        create_list(:user, evaluator.users_count, organizations: [organization])
```

Comment:
Sorry, could you explain this a bit more? What's creating random users in the organization?

Having to edit this random unrelated line of code that's been fine for 2 months worries me a little bit